src/main/java/utility/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package utility;
+
+public interface AreaCalculator {
```

Comment:
Interface names should not be verbs. Instead of 'AreaCalculator', a noun such as 'AreaComputable' or 'AreaProvider' would be more appropriate, reflecting the capability or property of the classes that will implement this interface.